docs/int/quickstart/advanced/quickstart-builder-api.md

```diff
@@ -27,79 +27,47 @@ charon run --builder-api
 
-Configuring the Teku validator client with Charon follows exactly the same process as their [official guide](https://hackmd.io/@StefanBratanov/BkMlo1RO9) with 2 further conditions.
+The validator client must be set up to use the `--validators-proposer-config` [flag](https://docs.teku.consensys.net/reference/cli#validators-proposer-config) with a value equal to `http://$CHARON_ENDPOINT:3600/teku_proposer_config`.
 
-- First the validator client must be set up to use the `proposerConfig.json` structure. This involves including the [`--validators-proposer-config`](https://docs.teku.consensys.net/en/latest/Reference/CLI/CLI-Syntax/#validators-proposer-config) flag on the validator client.
-- Second the `--validators-proposer-config` flag must be equal to `http://$CHARON_ENDPOINT:3600/teku_proposer_config`
-
-With these 2 conditions in places Charon validators will be able to register to the builder network, submit blinded block and gain a share of the MEV profits.
+Once the flag is set up, Obol distributed validators will be able to register to the builder network, submit blinded beacon blocks and gain a share of the MEV profits.
 
 ### Lighthouse Validator Client
 
-For Lighthouse we are currently waiting on the following [PR](https://github.com/sigp/lighthouse/pull/3445) to be merged into their unstable branch to enable compatability, please review the PR's status.
-
-If the PR has been merged Lighthouse can be used with Charon. You must use the [unstable branch / container image](https://hub.docker.com/r/sigp/lighthouse/tags) to have access to the changes that enable compatibility.
+For Lighthouse, we are currently waiting on the following [PR](https://github.com/sigp/lighthouse/pull/4306) to be merged into their unstable branch to enable compatability, please review the PR's status.
 
-Configuring the Lightouse validator client with Charon follows exactly the same process as their [official guide](https://lighthouse-book.sigmaprime.io/builders.html) with some additional conditions.
+## Run MEV Boost
 
-- The validator client must be set up to use a custom [validator_definitions.yml](https://lighthouse-book.sigmaprime.io/validator-management.html).
-- The flag `--builder-registration-timestamp-override` must be set and the assigned value must be the same across all validator clients.
-- The custom validator_definitions.yml must be placed in the `--validators-dir` of lighthouse.
-- The custom validator_definitions.yml must follow the structure below, where `voting_public_key` is the pubkey share on the validator client and `builder_pubkey_override` is the associated aggregate pubkey the network will find. You can find these pubkey share to aggregate pubkey mappings in the `cluster-lock.json` file created during the DKG process.
+If you want to run [mev-boost](https://boost.flashbots.net/) and if you are already running the [cluster](docs/int/quickstart/group/index.md), you can follow these steps:
```

Comment:
internal link broken?

Reply:
Full path instead of relative link again

Reply:
I changed this section entirely. Reason being that you can't run mev-boost with the current state of CDVN since we use lodestar VC which doesn't support `builder-api` with charon.

So, we need to wait till next release to fully support this feature.